contracts/Comptroller.sol

```diff
@@ -1003,7 +1056,16 @@ contract Comptroller is ComptrollerV8Storage, ComptrollerInterfaceG2, Comptrolle
       * @param newBorrowCaps The new borrow cap values in underlying to be set. A value of 0 corresponds to unlimited borrowing.
       */
     function _setMarketBorrowCaps(VToken[] calldata vTokens, uint[] calldata newBorrowCaps) external {
-        ensureAdminOr(borrowCapGuardian);
+       
+        // NOTE: previous code restricted this function with
+        // msg.sender == admin || msg.sender == borrowCapGuardian
+        // Please consider adjusting deployment script before Testnet
```

Comment:
Let's remove this comment so that we don't scare people :)
Our borrowCapGuardian on mainnet is address(0) anyway